runtime/pack/dist/opt/netrw/plugin/netrwPlugin.vim

```diff
@@ -148,43 +148,6 @@ function! s:VimEnter(dirname)
     endif
 endfunction
 
-" }}}
-" NetrwStatusLine: {{{
-
-function! NetrwStatusLine()
```

Comment:
This version is in autoload/netrw.vim so should be fine. However, I'd recommend to use `:unlet!` as it is used here.

Reply:
Do you mean having a bang at in the function stament?